src/js/utils/dispatch.js

```diff
@@ -6,15 +6,15 @@
  * @param {Function} listener - the function to be executed
  * @returns {Object} - containing a <tt>trigger()</tt> method for executing the listener, and an <tt>off()</tt> method for detaching it
  */
-module.exports = function dispatch (element, eventTypes, listener) {
+module.exports = function dispatch (element, eventTypes, listener, options) {
```

Comment:
If it's not being used then I strongly recommend taking it out. Unused code infrastructure is great for confusing people, but not much else. Especially this kind of change, which is very easy to add when we need it.